components/centraldashboard/frontend/main-page.html

```diff
@@ -170,13 +195,20 @@
                     {text: 'Compare run output', icon: 'donut-large'},
                     {text: 'Share solution on AI Hub', icon: 'donut-large'},
                 ]},
+                quickLinks: {type: Array, value: [
+                    {text: 'Open docs', link: 'https://www.kubeflow.org/docs/started/getting-started/'},
+                    {text: 'Open Github', link: 'https://github.com/kubeflow/kubeflow'},
+                ]},
                 sidebarItemIndex: {type: Number, value: 0},
                 primaryViewIndex: {type: Number, value: 0},
                 homeOrIframeViewIndex: {type: Number, value: 0},
                 url: {type: String, value: ""},
+                buildVersion: {type: String, value: "1.0"},
             }}
-            static get observers() {return [
-            ]}
+            openQuickLink(e) {
+                const {link} = e.model.item
+                ;(a => {a.href = link;a.target='_blank';a.click()})(document.createElement('a'))
```

Comment:
move semi-colon

Why make this an anonymous function rather than just `const anchor = document.createElement('a');` etc?

Reply:
Brevity, like in python:
```js
with object as o:
   // use o
```
Semi-colon here is required, just like with any js line that starts with `[` or `(`

I'll change it, probably best to write readable code :)